cruise.umple/src/Umple_Code.ump

```diff
@@ -778,7 +778,38 @@ class Method
 	  }
 	  this. comments = another.getComments();
   }
-
+  public boolean existsAlreadyInClass(final UmpleClassifier umpleClass) {
+	if (umpleClass != null && umpleClass.getMethods() != null) {
+		for (final Method method : umpleClass.getMethods()) {
+			// Ensure the method names match
+			if (!method.getName().equals(this.name)) {
+				continue;
+			}
+			
+			// Check if parameters match
+			boolean isSame = true;
+			if (method.getMethodParameters() != null && this.methodParameters != null) {
+				for (final MethodParameter parameter : this.methodParameters) {
+					boolean parameterExists = false;
+					for (final MethodParameter otherParameter : method.getMethodParameters()) {
+						if (parameter.getName().equals(otherParameter.getName()) && parameter.getType().equals(otherParameter.getType())) {
+							parameterExists = true;
+							break;
+						}
+					}
+					
+					if (!parameterExists) {
+						isSame = false;
+					}
+				}
+			}
+			
+			return isSame;
```

Comment:
This loop checks for the case whether both methods in question have parameters or not, if they do:

1. Loop through each parameter in MethodA
2. Compare each parameter in MethodA with **ALL** parameters in MethodB, if there is a single match, we are good to move onto the next parameter for MethodA
3. Only in the case where the parameter in MethodA was not found at all in MethodB's parameters list will it trigger the conditional that sets the isSame flag to false

Hopefully I did not make a mistake in the logic here, but it seems that this should work fine for determining whether a method is the same as another method by checking the method name, as well as all of its parameters if it has any. 

I went with a "prove me wrong" approach with this loop, so only in the case where a parameter is not found will it return false for isSame